### DISCUSSION TITLE
關注社會議題的管道

### DISCUSSION THREAD

**Discussion by forestdew507** [1101-GS4719-C/Python-Community-GS4719-C#2683] [General]
「挑選值得信任的媒體，像報導者、轉角國際、中央社，以及韓半島新聞平台粉專。」在敏迪的貼文最後看到這句，想到好奇大家都透過哪些管道與社會連結 ? 

個人覺得，雖然點開 google 很快，但因為演算法，總是只會看到特定幾類的文章，沒辦法協助我更全面的與社會議題連結，所以或許直接到值得信任的自媒體主頁會是關注時事更好的選項

**Comment by wcyeee** [comment]
我自己是看中央社的報導！感覺中央社比較常報一些政治或是國際議題這樣（也都蠻中立的感覺嗎？）然後聯合報我自己也有在看，但感覺比較偏向日常生活，不是社會議題比較多。我覺得以上兩個看起來都蠻舒服的，然後中央社比較偏向有比較多社會議題的樣子

**Comment by Elaine4488** [comment]
我會看關鍵評論網！他們的 YouTube 頻道每週都會整理國際新聞，還會有一些深入的特別報導。比起一般的文字跟圖片報導，我很喜歡他們的圖表呈現，不僅有資訊量，還很有設計感，讓人讀起來不會覺得枯燥，能夠更輕鬆的消化這些資訊。
